### ISSUE TITLE
Adding a listing to see sessions or 3 last used files?

### ISSUE THREAD

**Issue by thepenguinthatwants** [goolord/alpha-nvim#72]
Also recent used sessions or seeing 3 final files that has being used could be cool.
Is this possible with alpha?

Like

g.startify_custom_indices = {'a', 's', 'd', 'f', 'g', 'h', 'i'}
g.startify_bookmarks
g.startify_session_dir = '~/.config/nvim/session'

I've actually gotten attached to use dashboard view, but some tools from startify has being itching to get.

**Comment by goolord** [comment]
yeah, the startify theme actually exports the 'mru' buttons that you're describing

https://github.com/goolord/alpha-nvim/blob/main/lua/alpha/themes/startify.lua#L241

you might have to write a little code to make it look how you want

sessions support is a WIP

**Comment by rockyzhang24** [comment]
At least it would be better to have a list showing all sessions and load a session when select it.
